fire-hose-management-frontend/src/components/Departments/DepartmentOverlay.jsx

```diff
@@ -1,158 +1,204 @@
+    const [postalCode, setPostalCode] = useState(initialPostalCode)
+    const [location, setLocation] = useState(initialLocation)
+    const [country, setCountry] = useState(initialCountry)
+    const [gender, setGender] = useState(initialGender)
+    const [firstname, setFirstname] = useState(initialFirstname)
+    const [lastname, setLastname] = useState(initialLastname)
+    const [mail, setMail] = useState(initialMail)
+    const [sendConfirmationMail, setSendConformationMail] = useState(initialSendConfirmationMail)
 
-    </Overlay>
-  );
+    const dtoBuilder = (departmentName, street, houseNumber, postalCode, location, country, firstname, lastname, mail, sendConfirmationMail) => {
+        return {
+            departmentName: departmentName,
+            contact: {
+                gender: gender,
+                firstname: firstname,
+                lastname: lastname,
+                mail: mail,
+            },
+            street: street,
+            houseNumber: houseNumber,
+            postalCode: postalCode,
+            location: location,
+            country: country,
+            sendConfirmationMail: sendConfirmationMail
+        }
+    }
+
+
+    return (<Overlay
+        onClose={() => handleClose()}
+        headerContent={<h2>{headline}</h2>}
+        size={"m"}>
+        <ValidatorForm
+            onSubmit={() => submitBtnFunction(dtoBuilder(departmentName, street, houseNumber, postalCode, location, country, firstname, lastname, mail, sendConfirmationMail))}>
+            <Grid container spacing={3}>
+                <Grid item xs={12}>
+                    <CTextField
+                        hasAutofocus={true}
+                        label={"Abteilung"}
+                        value={departmentName}
+                        onChange={(event) => setDepartmentName(event.target.value)}
+                        validators={["required"]}
+                        errorMessages={["Die Eingabe einer Abteilung ist erforderlich!"]}
+                    />
+                </Grid>
+                <Grid item md={6}>
+                    <CTextField
+                        label={"Straße"}
+                        value={street}
+                        onChange={(event) => setStreet(event.target.value)}
+                        validators={["required"]}
+                        errorMessages={["Die Eingabe eines Straßennamens ist erforderlich!"]}
+                    />
+                </Grid>
+                <Grid item md={6}>
+                    <CTextField
+                        label={"Hausnummer"}
+                        value={houseNumber}
+                        onChange={(event) => setHouseNumber(event.target.value)}
+                        validators={["required"]}
+                        errorMessages={["Die Eingabe einer Hausnummer ist erforderlich!"]}
+                    />
+                </Grid>
+                <Grid item md={4}>
```

Comment:
Codacy has a fix for the issue: [Delete `······`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
          <Grid item md={4}>
```